### PR TITLE
Add mapping for aria-braillelabel and aria-brailleroledescription

### PR DESCRIPTION
**Pull request by spectranaut**
https://github.com/w3c/aria/issues/1331

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/pull/134.html" title="Last updated on Aug 25, 2022, 6:36 PM UTC (5055790)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/134/1d4e8a5...5055790.html" title="Last updated on Aug 25, 2022, 6:36 PM UTC (5055790)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2399,6 +2399,36 @@ <h4>Not Mapped</h4>
...
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.brailleroledescription</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleRoleDescription</span>
+				</td>
+			</tr>
```
Comment (spectranaut) [inline index.html:2431]: I wonder if I should add a second entry for aria-brailleroledescription:

aria-roledescription is empty or whitespace characters
- IA2: not mapped
- UIA: not mapped
- ATK: not mapped
- AXAPI: not mapped

Comment (spectranaut) [reply index.html:2431]: or maybe it can just be a note?

Comment (aleventhal) [reply index.html:2431]: A note is fine. We also need to not expose when there is no aria-roledescription.

File: index.html [hunk 2]
```diff
@@ -2399,6 +2399,36 @@ <h4>Not Mapped</h4>
...
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.braillelabel</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleLabel</span>
```
Comment (aleventhal) [inline index.html:2414]: @cookiecrook are the Mac mappings correct?

**Review by aleventhal — APPROVED** [review]
The IA2/ATK mappings look good.
We should ask Travis Leithead for feedback on the UIA mappings, file an issue at crbug.com for implemention them, and maybe ask MS to consider it as a feature request for Narrator.

File: index.html (hunk 2 above)
Comment (cookiecrook) [reply index.html:2414]: Yes https://github.com/WebKit/WebKit/blob/main/Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm#L496

File: index.html [hunk 3]
```diff
@@ -2399,6 +2399,51 @@ <h4>Not Mapped</h4>
...
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleRoleDescription</span>
+				</td>
+			</tr>
+			<tr id="ariaBrailleroledescriptionUndefined">
+				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a> is undefined, contains only whitespace characters, or the empty Braille pattern: dots-0 (U+2800)</th>
```
Comment (cookiecrook) [inline index.html:2433]:
```suggestion
				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a> is undefined the empty string</th>
```

per discussion in PR #135 and [related WG decision](https://www.w3.org/2022/08/11-aria-minutes#t05), should we remove all the references to whitespace here?

Comment (spectranaut) [reply index.html:2433]: yes I'll fix now, thanks! :)

**Review by jnurthen — APPROVED** [review]

**Comment by spectranaut** [comment]
> The IA2/ATK mappings look good. We should ask Travis Leithead for feedback on the UIA mappings, file an issue at crbug.com for implemention them, and maybe ask MS to consider it as a feature request for Narrator.

I was just checking to see if this bug was filed, and I see it is:
https://bugs.chromium.org/p/chromium/issues/detail?id=1352278&q=aria-braille&can=2

**Pull request merged**